cd/terraform/instance.tf

```diff
@@ -0,0 +1,35 @@
+data "openstack_compute_keypair_v2" "keypair" {
+  name = "basegun-keypair-tf"
+}
+
+data "openstack_networking_network_v2" "network" {
+  name     = "Ext-Net"
+}
+
+data "openstack_images_image_v2" "debian" {
+  name        = var.image
+  most_recent = true
+}
+
```

Comment:
If it's not too complex (or too costy), we would appreciate have a s3 volume mounted as well. I don't know how that works, I would be happy to discuss about it

Reply:
Yes, it's can be a good issue! I don't know how that is possible.